tests/wpt/webgl/meta/MANIFEST.json

```diff
@@ -8719,7 +8719,7 @@
      []
     ],
     "npot-video.theora.ogv": [
-     "4458678fbfd81997ee5eff54f8b5e7fbd855264a",
+     "e2e7a25177072b19cb21be65c57d3b60c48cc9ce",
```

Comment:
Also this should be undone, I would suggest just manually re-adding the old hashes. And then we need to re-add the binary files that have been removed below.

Reply:
Okay, I'll diff the changes to the Manifest.json and copy the old hashes in to replace the changed hashes.